src/disco/events/fd_circbuf.h

```diff
@@ -0,0 +1,65 @@
+#ifndef HEADER_fd_src_disco_events_fd_circbuf_h
+#define HEADER_fd_src_disco_events_fd_circbuf_h
+
+#include "../fd_disco_base.h"
+
+#define FD_CIRCBUF_ALIGN (4096UL)
+
+struct __attribute__((aligned(FD_CIRCBUF_ALIGN))) fd_circbuf_private {
+  ulong cnt;
+  ulong head;
+  ulong tail;
+  ulong size;
+
+  struct {
+    ulong drop_cnt;
+  } metrics;
+};
```

Comment:
```suggestion
  /* padding out to 4k here ... */
};
```
I missed the fact this struct is so large and was confused about alignment.